Automattic/vip-go-mu-plugins
# Update Jetpack 11.2 to release

## Description
Update Jetpack 11.2 to release

## Changelog Description
### Plugin Updated: Jetpack 11.2

We upgraded Jetpack 11.2 to release.

## Checklist

## Steps to Test
1. Check out PR.
1. Pin Jetpack version at 11.2
1. Go to `wp-admin` > `Jetpack`
1. Verify Jetpack version is 11.2.
